docs/settings/alert-action-settings.asciidoc

```diff
@@ -71,6 +71,13 @@ a|`xpack.actions.`
   +
   As an alternative to setting both `xpack.actions.proxyRejectUnauthorizedCertificates` and `xpack.actions.rejectUnauthorized`, you can point the OS level environment variable `NODE_EXTRA_CA_CERTS` to a file that contains the root CAs needed to trust certificates.
 
+| `xpack.actions.maxResponseContentLength` {ess-icon}
+  | Defines the max size of the http response content in bytes allowed in node.js. Defaults to 1000000 (1MB).
```

Comment:
I feel like we should be explicit that this applies to the requests made by actions to external resources.

```suggestion
| `xpack.actions.maxResponseContentLength` {ess-icon}
  | Specifies the max size of the http response content in bytes allowed in any request initiated by an action to an external resource. Defaults to 1000000 (1MB).
```

Reply:
@gchaps I suspect I'm being too verbose here (as usual). Would love your thoughts.